test/chat_api/conversations_test.exs

```diff
@@ -85,6 +86,27 @@ defmodule ChatApi.ConversationsTest do
       assert_raise Ecto.NoResultsError, fn -> Conversations.get_conversation!(conversation.id) end
     end
 
+    test "delete_conversation/1 deletes the conversation if associated slack_conversation_threads exist",
+         %{conversation: conversation} do
+      assert {:ok, %Conversation{} = conversation} =
+               Conversations.create_conversation(valid_create_attrs())
+
+      slack_conversation_thread_attrs = %{
+        slack_channel: "some slack_channel",
+        slack_thread_ts: "some slack_thread_ts",
+        conversation_id: conversation.id,
+        account_id: valid_create_attrs().account_id
+      }
+
+      assert {:ok, %SlackConversationThread{}} =
+               SlackConversationThreads.create_slack_conversation_thread(
+                 slack_conversation_thread_attrs
+               )
+
+      assert {:ok, %Conversation{}} = Conversations.delete_conversation(conversation)
```

Comment:
can we also assert that the created SlackConversationThread is also deleted?

Reply:
Thank you! This should be good now :+1: